### PR TITLE
Updated image download deploy

### PR DESCRIPTION
**Pull request by oscarkraemer**
Updated image_dl_test_deploy.sh to reflect image_create.sh

### CODE REVIEW

File: scripts/image_dl_test_deploy.sh [hunk 1]
```diff
@@ -1,11 +1,21 @@
 #!/bin/bash
 # Download cloud image, push it to glance and create test instance
 
-if [[ $(dirname $0) == *epouta* ]]; then
-source $(dirname $0)/image_epouta_constants.sh
+echo "=== "$(date)
+
+# Expiting the file location to be:
```
Comment (martbhell) [inline scripts/image_dl_test_deploy.sh]: typo

File: scripts/image_dl_test_deploy.sh [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
+# /SOME/PATH/diskimage-builder-scripts.${openstack_environment}/image_${openstack_environment}_constants.sh
+openstack_environment="$( echo $(dirname $0) | sed 's/^.*diskimage-builder-scripts\.//g' |cut -d '/' -f 1)"
```
Comment (martbhell) [inline scripts/image_dl_test_deploy.sh:8]: Very nice. Seems to work when running. Ran the line in the cronjob manually from $HOME of imgbuilder. Also adjusted the crontab and let a centos7 build run and that worked too.

**Review by martbhell — CHANGES_REQUESTED** [review]
Small typo other than that very nice :)

File: scripts/image_dl_test_deploy.sh (hunk 1 above)
Comment (oscarkraemer) [reply scripts/image_dl_test_deploy.sh]: Fixed the typo also in image_create.sh

**Comment by oscarkraemer** [comment]
Update typo also in image_create.sh

**Review by martbhell — APPROVED** [review]

**Pull request merged**